lighthouse-core/scripts/legacy-javascript/main.js

```diff
@@ -1,5 +1,5 @@
-// @ts-nocheck
 /* eslint-disable */
+// @ts-nocheck
```

Comment:
how was this related?

Reply:
Jsdoc rule failed otherwise

Reply:
lol well I assumed so, but I was wondering if there's something lurking here. This file doesn't have a license. Shouldn't that be impossible now? Is this a bug in the rule?

Reply:
Yes i think it's a bug in the rule, it didn't respect `eslint-disable` unless it was the first comment.

It's working correctly that there is no license error check for this file since eslint should be disabled.